includes/footer.php

```diff
@@ -94,39 +87,34 @@
-          <li><a href="/docs/usage/reference_genomes">Reference genomes</a></li>
-          <li><a href="/docs/usage/data_management">Data Management</a></li>
-          <li><a href="/docs/usage/troubleshooting">Troubleshooting</a></li>
-          <li><a href="/docs/usage/nextflow">Nextflow resources</a></li>
+          <li><a href="https://nf-co.re/modules">nf-core modules</a></li>
+          <li><a href="https://nf-co.re/tools">nf-core tools</a></li>
+          <li><a href="https://nf-co.re/docs/usage/introduction">Getting started</a></li>
+          <li><a href="https://nf-co.re/docs/usage/installation">Installation</a></li>
+          <li><a href="https://nf-co.re/docs/usage/configuration">Pipeline configuration</a></li>
+          <li><a href="https://nf-co.re/docs/usage/offline">Running offline</a></li>
+          <li><a href="https://nf-co.re/docs/usage/usage_tutorials">Usage tutorials</a></li>
+          <li><a href="https://nf-co.re/docs/usage/data_management">Data Management</a></li>
+          <li><a href="https://nf-co.re/docs/usage/troubleshooting">Troubleshooting</a></li>
+          <li><a href="https://nf-co.re/docs/usage/nextflow">Nextflow resources</a></li>
         </ul>
       </div>
       <div class="col-sm-6 col-lg-3 mb-3 d-print-none">
         <h5>For Authors</h5>
         <ul class="list-unstyled">
-          <li><a href="/docs/contributing/guidelines">Guidelines</a></li>
-          <li><a href="/docs/contributing/adding_pipelines">Adding a new pipeline</a></li>
-          <li><a href="/docs/contributing/modules">DSL2 modules</a></li>
-          <li><a href="/docs/contributing/subworkflows">DSL2 subworkflows</a></li>
-          <li><a href="/docs/contributing/release_checklist">Release checklist</a></li>
-          <li><a href="/docs/contributing/training_guide">Training guide</a></li>
-          <li><a href="/tools-docs">Lint error codes</a></li>
-          <li><a href="/docs/contributing/sync">Template synchronisation</a></li>
-          <li><a href="/docs/contributing/developer_tutorials">Developer tutorials</a></li>
-          <li><a href="/docs/contributing/editor_plugins">Code editor plugins</a></li>
-          <li><a href="/docs/contributing/design_guidelines">Graphic design guidelines</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/modules">DSL2 modules</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/subworkflows">DSL2 subworkflows</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/training_guide">Training guide</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/sync">Template synchronisation</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/developer_tutorials">Developer tutorials</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/editor_plugins">Code editor plugins</a></li>
+          <li><a href="https://nf-co.re/docs/contributing/design_guidelines">Graphic design guidelines</a></li>
```

Comment:
Do you want to change these internal links to external links? 

It is fine, the user can always get the latest if we don't pull their changes

Reply:
@gq1 : alternatively, what about modifying `_routes.php` so that all links to `/docs` are redirected to `https://nf-co.re/docs` ?

Reply:
Will we still be able to link to our `docs/` once the `routes.php` is updated?